src/api/ProjectsApi.js

```diff
@@ -14,13 +14,11 @@ export default class ProjectsApi {
   constructor(apiClient) {
     this.apiClient = apiClient || ApiClient.instance;
     this.retrieveProject = this.retrieveProject.bind(this);
-    this.retrieveProjectWithHttpInfo = this.retrieveProjectWithHttpInfo.bind(
-      this
-    );
+    this.retrieveProjectWithHttpInfo =
+      this.retrieveProjectWithHttpInfo.bind(this);
     this.retrieveProjects = this.retrieveProjects.bind(this);
-    this.retrieveProjectsWithHttpInfo = this.retrieveProjectsWithHttpInfo.bind(
-      this
-    );
+    this.retrieveProjectsWithHttpInfo =
+      this.retrieveProjectsWithHttpInfo.bind(this);
```

Comment:
It looks like there was a problem with my prettier settings yesterday and the code formatting hadn't run properly. Sorry about the noise here!